spec/spec_helper.rb

```diff
@@ -17,6 +17,8 @@
 require 'coveralls'
 Coveralls.wear!('rails')
 
+require File.join(File.dirname(__FILE__), 'support/factory_girl.rb')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.